teammapper-backend/src/map/services/maps.service.ts

```diff
@@ -37,6 +37,10 @@ export class MapsService {
     })
   }
 
+  async updateLastAccessed(uuid: string, lastAccessed = new Date()) {
+    await this.mapsRepository.update(uuid, { lastAccessed })
```

Comment:
`return this.mapsRepository.update(uuid, { lastAccessed })` and add the return type, e.g. Promise<void> (not sure what update returns)

Reply:
I think update returns something like UpdateResult which includes the affected row. Question is if we actually need that return value? When using deleteMap() (I think) we just call `this.mapsRepository.delete` and return void instead.